packages/react-expandable-table/src/styles.js

```diff
@@ -0,0 +1,223 @@
+    z-index: 1;
+  }
+`;
+Row.defaultProps = {
+  theme: themes.dark,
+};
+
+export const ExpandableContent = styled.div`
+  box-shadow: inset 0 -2px 2px -2px ${props => props.theme.colors.gray5};
+  background-color: ${props => props.theme.colors.gray7};
+`;
+ExpandableContent.defaultProps = {
+  theme: themes.dark,
+};
+
+export const List = styled(GroupedVirtuoso, {
+  shouldForwardProp: prop =>
+    ['group', 'item', 'groupCounts'].includes(prop) || isPropValid(prop),
+})`
+  color: ${props => props.theme.primary};
+  background-color: ${props => props.theme.colors.gray7};
+  border: 2px solid ${props => props.theme.colors.gray6};
+  border-radius: 2px;
+  max-height: ${props => (props.maxHeight ? `${props.maxHeight}px` : 'auto')};
+`;
+List.defaultProps = {
+  theme: themes.dark,
+};
+
+export const Header = styled.header`
+  display: flex;
+  align-items: stretch;
+  background-color: ${props => props.theme.colors.gray7};
+  border-bottom: 1px solid ${props => props.theme.selected};
+  border-radius: 2px 2px 0 0;
+  padding: 0 ${props => props.theme.sizes.regular};
+  height: 40px;
+  font-weight: bold;
+
+  ${Container} {
+    font-weight: regular;
+  }
+`;
+Header.defaultProps = {
+  theme: themes.dark,
+};
+
+export const AngleButton = styled.button`
+  all: unset;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transform: rotate(${props => (props.open ? 180 : 0)}deg);
+  transition: transform 0.5s ease-out;
+  ${textStyles('xlarge')};
+  border-radius: 50%;
+  width: 20px;
+  height: 20px;
+  &:focus-visible {
+    box-shadow: 0 0 2px 2px ${props => props.theme.selected};
+  }
+`;
+AngleButton.defaultProps = {
+  theme: themes.dark,
+};
+
+export const Ellipsis = styled(Text)`
+  display: block;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  margin-top: -2px;
+  cursor: pointer;
+  user-select: none;
+  color: ${props =>
+    props.inactive ? props.theme.colors.gray5 : props.theme.primary};
+  &:focus:not(:focus-visible) {
+    outline: 0;
+  }
+`;
+Ellipsis.defaultProps = {
+  theme: themes.dark,
+  tabIndex: -1,
+};
+
+export const InfoIcon = styled(
+  React.forwardRef((props, ref) => (
+    <button type="button" {...props} ref={ref}>
+      <Icon name="question_circle" />
+    </button>
+  ))
+)`
+  all: unset;
+  cursor: pointer;
+  color: ${props => props.theme.colors.gray5};
+  color: ${props => props.theme.prmary};
+  margin-left: ${props => props.theme.sizes.small};
+
+  &:hover,
+  &:focus-visible {
+    color: ${props => props.theme.primary};
+  }
+`;
+InfoIcon.defaultProps = {
+  theme: themes.dark,
+};
+
+const SortAsc = styled(props => <Icon name="sort_asc" {...props} />)``;
+const SortDesc = styled(props => <Icon name="sort_desc" {...props} />)``;
+const ACTIVE_ARROW = props =>
+  props.sort == null ? SortDesc : props.sort === 'desc' ? SortAsc : '.';
+const NEXT_ARROW = props =>
+  props.sort === 'desc' ? SortDesc : props.sort === 'asc' ? SortAsc : '.';
```

Comment:
Preference for clarity over simplicity.

Reply:
These are "dynamic" selectors, given the `sort` property they return the selector we want to target.

Reply:
I just realized I did invert the names, I fixed it. I guess it's clearer now :-)

Reply:
It like how elegant it is but it is hard for a reviewer to catch that mistake.

Reply:
I'm open to alternatives